### ISSUE TITLE
[Feature request] Possibility to disable horizontal bouncing in ion-slide-box 

### ISSUE THREAD

**Issue by elacca** [driftyco/ionic#1172]
Hi!

It would be good, if the horizontal bouncing in ion-slide-box could be disabled optionally (first & last slide).

Right now I've disabled this bouncing functionality by changing the lines 6226-6233 in ionic.bundle.js in 1.0.0-beta1 from this : 

```
delta.x =
              delta.x /
                ( (!index && delta.x > 0               // if first slide and sliding left
                  || index == slides.length - 1        // or if last slide and sliding right
                  && delta.x < 0                       // and if sliding at all
                ) ?
                ( Math.abs(delta.x) / width + 1 )      // determine resistance level
                : 1 ); 
```

to this:

```
delta.x = (!index && delta.x > 0 || (index == slides.length - 1 && delta.x < 0)) ? 0 : delta.x;
```

**Comment by PhilHuot** [comment]
What is the status of this issue?

**Comment by mlynch** [comment]
Thanks! This was just merged into master and will be in 1.2: f617a9d181953354057879e0e54fa0a4e0aa6169

**Comment by DevNIX** [comment]
What's the usage? I'm trying to use it with the [tabbedSlideBox](https://github.com/saravmajestic/ionic) plugin (the horizontal bounce glitches on iOS and is not disabled on Android like the rest of components) but it doesn't work.

It could be a problem of my plugin?
